### ISSUE TITLE
`apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+`   when  helm install greater than v1.16+

### ISSUE THREAD

**Issue by xiaohui** [openkruise/kruise#590]
**What happened**:
warning messages  display when  helm install on the kubernetes's version is   greater than v1.16
```
W0409 12:13:24.452188   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.458981   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.466526   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.473538   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.482179   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.508830   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
W0409 12:13:24.517170   18187 warnings.go:70] apiextensions.k8s.io/v1beta1 CustomResourceDefinition is deprecated in v1.16+, unavailable in v1.22+; use apiextensions.k8s.io/v1 CustomResourceDefinition
```

**What you expected to happen**:
no warning messages

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kruise version:
v0.8.1 
- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.4-dirty", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"dirty", BuildDate:"2021-03-15T09:58:13Z", GoVersion:"go1.16.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.5", GitCommit:"e338cf2c6d297aa603b50ad3a301f761b4173aa6", GitTreeState:"clean", BuildDate:"2020-12-09T11:10:32Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
```

- Kernel (e.g. `uname -a`):
- Install tools:
helm
- Others:

**Comment by FillZpp** [comment]
@xiaohui To make Kruise compatible with Kubernetes version lower than 1.16, we had to use CRD v1beta1 currently.
But we do have a plan to bump CRD to v1 in future, maybe Kruise v1.0 release.

**Comment by FillZpp** [comment]
fixed by #743
